Translations/cs.lproj/preferences.strings

```diff
@@ -330,7 +330,7 @@
 "u2R-2w-aOA.title" = "Pro novou komprimaci, použít výchozí heslo";
 
 /* Class = "NSTextFieldCell"; title = "You can also use this password from the Finder Extension's menu"; ObjectID = "vWb-gb-JBD"; */
-"vWb-gb-JBD.title" = "Toto heslo lze také použít v kontextové nabídce Finderu";
+"vWb-gb-JBD.title" = "Toto heslo lze také použít v kontextové nabídce Finderu.";
```

Comment:
That's a good one

Reply:
This type of notes are consider as sentence, so it should end with punctuation (period, question or exclamation mark), so there should be period "." at the end also in English and all other languages.
In other places this rule is kept. See for example "File Access" or "Performance" tab in "Preferences".
![ScreenShot 2023-09-05 at 12 02 53](https://github.com/aonez/Keka/assets/45659268/022d9d3d-647f-46b5-85a0-e53abe01a61f)

Reply:
Oh it's already done! Thank you